modules/performanceplatform/manifests/proxy_vhost.pp

```diff
@@ -53,6 +53,13 @@
 
   $graphite_servername = regsubst($servername, '\.', '_', 'G')
 
+  if ($serveraliases != undef) {
+    validate_array($serveraliases)
+    $vhost_names = $serveraliases
+  } else {
+    $vhost_names = [$servername]
```

Comment:
I'm going to merge this, but wonder if we should consider merging `$servername` and `$serveraliases`, or failing if both are passed, as as it currently stands if both were passed only `$serveraliases` would be picked up, which could be confusing...